modules/sc-mesh-secure-deployment/src/nats/scripts/cli_visual_subscribe.py

```diff
@@ -24,13 +24,12 @@ async def subscribe_handler(msg):
         subject = msg.subject
         reply = msg.reply
         data = json.loads(msg.data.decode())
-        print("Received a message on '{subject} {reply}': {data}".format(
-          subject=subject, reply=reply, data=data))
+        print(f"Received a message on '{subject} {reply}': {data}")
 
     try:
-        await client.connect(nc, reconnected_cb=reconnected_cb,
+        await client.connect(nc, recon_cb=reconnected_cb,
                              closed_cb=closed_cb,
-                             max_reconnect_attempts=-1)
+                             max_recon_attempts=-1)
```

Comment:
While there was room for improvement in the "reconnect" vs "reconnected" naming, it might be worth to not shorten so much down to "recon" which is likely to get confused with "reconnaissance" :)

Reply:
fixed in https://github.com/tiiuae/mesh_com/commit/b67d5f9e3c84fcec3732cb04a35297e5acbe08f4